package.json

```diff
@@ -187,12 +187,15 @@
     "cozy-notifications": "0.5.0",
     "cozy-pouch-link": "6.64.7",
     "cozy-realtime": "3.3.1",
+    "cozy-scripts": "3",
     "cozy-stack-client": "8.7.0",
     "cozy-ui": "29.1.0",
     "d3": "5.11.0",
     "date-fns": "1.30.1",
     "detect-node": "2.0.4",
     "element-scroll-polyfill": "1.0.1",
+    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-react-hooks": "^3.0.0",
```

Comment:
It should come with https://github.com/cozy/cozy-libs/blob/master/packages/eslint-config-cozy-app/package.json#L27 non ?

Reply:
I figured I would remove dependencies managed via cozy-scripts in a later PR, but we may as well do it now : https://github.com/cozy/cozy-banks/pull/1793/commits/4b1bb1a15f6282a5d392b8f9abc76065605bd21f